charts/pega/templates/_pega-deployment.tpl

```diff
@@ -85,6 +85,10 @@ spec:
         # Node type of the Pega nodes for {{ .name }}
         - name: NODE_TYPE
           value: {{ .nodeType }}
+        - name: NODE_TIER
+          value: {{ .tierName }}
+        - name: NODE_SETTINGS
+          value: Pega-UIEngine/cloud/isNodeAgnosticAdminStudio=true;Pega-IntegrationEngine/EnableRequestorPools=false;
```

Comment:
For node settings that we want to always be present in the Docker image, we should probably just add them [here](https://github.com/pegasystems/docker-pega-web-ready/blob/master/tomcat-conf/Catalina/localhost/prweb.xml)

Reply:
If you are going to force in NODE_SETTINGS you need to allow customization, the SDE uses SDE specific NODE_SETTINGS

Reply:
Currently we are setting NODE_SETTINGS as a custom env var in the SDE as per

```
tier:
  - name: my-tier
    custom:
      env:
        - name: MY_ENV_NAME
          value: MY_ENV_VALUE
```

Reply:
Ok, i misspoke, we actually customize by passing in a custom JAVA_OPTS value, but the principal is the same, if you are going to do this we need to be able to override

Reply:
Example from an SDE web pod

![Capture](https://user-images.githubusercontent.com/54679517/72904956-b805c600-3cfd-11ea-8164-5fc42b93f173.PNG)

Reply:
This has been removed.  Closing thread.